.github/actions/setup-podman/registries.conf

```diff
@@ -0,0 +1,6 @@
+unqualified-search-registries = ['docker.io']
```

Comment:
this is a template file, not the final registrues.conf.. perhaps we should name it like this to reflect that, e.g. `registries.conf.tpl` ?

Reply:
Good idea, thx @pedjak